src/profile/services.js

```diff
@@ -1,12 +1,32 @@
 import LoggingService from '@edx/frontend-logging';
+import pick from 'lodash.pick';
 
-import apiClient from '../config/apiClient';
-import { configuration } from '../config/environment';
-import {
-  camelCaseObject,
-  convertKeyNames,
-  snakeCaseObject,
-} from './utils';
+import { utils } from '../common';
+
+const { camelCaseObject, convertKeyNames, snakeCaseObject } = utils;
+
+let config = {
+  ACCOUNTS_API_BASE_URL: null,
+  CERTIFICATES_API_BASE_URL: null,
+  LMS_BASE_URL: null,
+  PREFERENCES_API_BASE_URL: null,
+};
+
+let apiClient = null;
+
+function validateConfiguration(newConfig) {
+  Object.keys(config).forEach((key) => {
+    if (newConfig[key] === undefined) {
+      throw new Error(`Service configuration error: ${key} is required.`);
+    }
+  });
+}
+
+export default function configure(newConfig, newApiClient) {
```

Comment:
So this is new - it's where you give the profile API service the data it needs.  It then validates that you've configured it properly.